### ISSUE TITLE
Allow to transmit paramaters in request body in Swagger API

### ISSUE THREAD

**Issue by matmu** [rstudio/plumber#852]
Requests with very long parameter values sent from the Swagger API return the error:

```R
414 Request-URI Too Large
```

This is because the URI is limited in length. To avoid this error, it would be awesome to have a tag, e.g. `@request_body` in the plumber R-script that tells the Swagger API to send the request parameters in the message body.

**Comment by thomasp85** [comment]
I don't believe there is a way to get the UI to describe the structure of the body currently
